src/main/java/seedu/duke/userinterface/command/Help.java

```diff
@@ -1,8 +1,46 @@
 package seedu.duke.userinterface.command;
 
+import seedu.duke.userinterface.CliMessages;
+
+/**
+ * This class is to be instantiated when the user enters the keyword "help". The arguments for the command are then
+ * passed into the Help object, and executing it will display help appropriately.
+ *
+ * @author neilbaner
+ * @version 0.1
+ */
+
 public class Help extends CliCommand {
+    // the arguments added by the user
+    String commandArgs;
+
+    /**
+     * Constructor for Help.
+     *
+     * @param commandArgs the arguments input by the user. Precondition: these must be all lowercase, and with all
+     *                    leading and trailing spaces stripped.
+     */
+    public Help(String commandArgs) {
+        this.commandArgs = commandArgs;
```

Comment:
There is commandParams in CliCommand class. Could it be better to just utilise them?